swri_roscpp/include/swri_roscpp/subscriber_impl.h

```diff
@@ -75,7 +75,8 @@ class SubscriberImpl
       min_latency_ = latency;
       max_latency_ = latency;
       total_latency_ = latency;
-    } else {
+    }
+    else {
```

Comment:
Recommend undoing these style changes to remain consistent.

Reply:
Any opinions on how that check should be handled if the header stamp isn't set?